### PR TITLE
Improve make PR (force build and help message)

### PR DESCRIPTION
**Pull request by sapk**
Add `make pr` to `make help` and force build assets before launch (mostly to update ui)

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -120,6 +120,7 @@ help:
 	@echo " - vet               examines Go source code and reports suspicious constructs"
 	@echo " - test              run unit test"
 	@echo " - test-sqlite       run integration test for sqlite"
+	@echo " - pr#PRID           build and start gitea from a PR with integration test data loaded"
```
Comment (jolheiser) [inline Makefile]: Maybe something similar to this instead? Or could use `[]` or `{}` I'm not sure what's preferred or best practice.
```suggestion
	@echo " - pr#<index>        build and start gitea from a PR with integration test data loaded"
```

File: contrib/pr/checkout.go [hunk 2]
```diff
@@ -247,7 +247,8 @@ func main() {
 			log.Fatalf("Failed to duplicate this code file in PR : %v", err)
 		}
 	}
-	time.Sleep(5 * time.Second)
+	//Force build of js, css, bin, ...
+	runCmd("make", "build")
```
Comment (guillep2k) [inline contrib/pr/checkout.go:251]: I'd consider running `clean-all` as well. I'm having problems when I'm careless about leftover files from other branches (e.g. temporary files that are marked as ignored remain in the tree, but in a different branch are considered added/untracked files).

Perhaps `clean-all` should be a dependency of `pr`?

File: Makefile [hunk 3]
```diff
@@ -120,6 +120,7 @@ help:
 	@echo " - vet               examines Go source code and reports suspicious constructs"
 	@echo " - test              run unit test"
 	@echo " - test-sqlite       run integration test for sqlite"
+	@echo " - pr#<index>        build and start gitea from a PR with integration test data loaded"
```
Comment (6543) [inline Makefile:124]: o nice: didn't know this exist

**Review by guillep2k — APPROVED** [review]

File: contrib/pr/checkout.go (hunk 2 above)
Comment (jolheiser) [reply contrib/pr/checkout.go:251]: Should build just be a dependency as well, rather than adding the call here?

**Review by 6543 — APPROVED** [review]

**Pull request merged**